### PR TITLE
Address review suggestions

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
Update the WebSocket, TCP, and UDP BBEs as per the review suggestions.

### CODE REVIEW

File: examples/websocket-retry-client/websocket_retry_client.bal [hunk 1]
```diff
@@ -3,15 +3,11 @@ import ballerina/websocket;
 
 public function main() returns error? {
     websocket:Client wsClient = check new("ws://localhost:9090/foo", {
-        // Set the maximum retry count to 20 so that it will try 20 times with the default interval of
+        // Set the maximum retry count to 5 so that it will try 5 times with the interval of
         // 1 second in between the retry attempts.
-        retryConfig: { maxCount: 20 }
+        retryConfig: { maxCount: 5, interval: 1 }
```
Comment (shafreenAnfar) [inline examples/websocket-retry-client/websocket_retry_client.bal:8]: lets format this.

Comment (Bhashinee) [reply examples/websocket-retry-client/websocket_retry_client.bal]: Addressed with [a84428d](https://github.com/ballerina-platform/ballerina-distribution/pull/3559/commits/a84428dae7cfa580e8f1df7482a6c2851cede29e)

File: examples/websocket-retry-client/websocket_retry_client.bal [hunk 2]
```diff
@@ -5,7 +5,10 @@ public function main() returns error? {
     websocket:Client wsClient = check new("ws://localhost:9090/foo", {
         // Set the maximum retry count to 5 so that it will try 5 times with the interval of
         // 1 second in between the retry attempts.
-        retryConfig: { maxCount: 5, interval: 1 }
+        retryConfig: {
+            maxCount: 5,
+            interval: 1
```
Comment (shafreenAnfar) [inline examples/websocket-retry-client/websocket_retry_client.bal:10]: shall we make the interval 5 seconds.

Comment (Bhashinee) [reply examples/websocket-retry-client/websocket_retry_client.bal]: Fixed with [08ef097](https://github.com/ballerina-platform/ballerina-distribution/pull/3559/commits/08ef0971d5918bbf5c48ed68414ff5006843425b)

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**